docs/samples/Microsoft.ML.Samples/Dynamic/FastTreeRegression.cs

```diff
@@ -35,16 +35,6 @@ public static void Example()
 
             // Get the trained model parameters.
             var modelParams = model.LastTransformer.Model;
-
-            // Let's see where an example with Parity = 1 and Induced = 1 would end up in the single trained tree.
-            var testRow = new VBuffer<float>(2, new[] { 1.0f, 1.0f });
-            // Use the path object to pass to GetLeaf, which will populate path with the IDs of th nodes from root to leaf.
-            List<int> path = default;
-            // Get the ID of the leaf this example ends up in tree 0.
-            var leafID = modelParams.GetLeaf(0, in testRow, ref path);
```

Comment:
Removed because our public APIs doesn't provide those functions.

Reply:
But you can still traverse through trees with new structure, right?

---
In reply to: [260562343](https://github.com/dotnet/machinelearning/pull/2753#discussion_r260562343) [](ancestors = 260562343)

Reply:
No, we cannot. @tfinley@gmail.com, do you think this feature is required in tree's public APIs?

---
In reply to: [261011054](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261011054) [](ancestors = 261011054,260562343)

Reply:
I think the trees should enable someone to explain what the tree is. I wouldn't make evaluation code part of the API.

---
In reply to: [261417026](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261417026) [](ancestors = 261417026,261011054,260562343)

Reply:
Sounds good. I feel evaluation module should be independent to the expression of the trained objects. Thanks you!

---
In reply to: [261734289](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261734289) [](ancestors = 261734289,261417026,261011054,260562343)